### PR TITLE
Fix Styling Bug

### PR DESCRIPTION
**Pull request by MrMagma**
This pull request fixes a styling "bug" where images for contests were either not centered above the contest control buttons or were far too large, and also addresses some minor code style issues.
### sass/main.scss
- Remove the `height` property on the `.contest-thumbnail` style so that the browser can maintain proper aspect ratio on images.
- Add a `max-width: 200px` property to the `.contest-thumbnail` style so that the thumbnails for contests will never exceed 200 pixels in width (their actual resolution).
## src/client/index.js
- Append the `img` tag to the `.center` div instead of the column so that it is centered with the contest controls.
## .gitattributes
- Create a .gitattributes file to ensure consistent Unix-style line-endings on the project no matter what git's global setting is.
### src/generalPurpose.js, gulpfile.js
- Fix inconsistent string quoting.
### build/src/_shared.js
- Add a `TODO`

### CODE REVIEW

**Comment by bytorbrynden** [comment]
Awesome! Thanks for the PR, I'll review it shortly.

File: src/generalPurpose.js [hunk 1]
```diff
@@ -11,10 +11,10 @@ module.exports = (function() {
             // Get the cookie with name cookie (return "" if non-existent)
             cookieName = cookieName + "=";
             // Check all of the cookies and try to find the one containing name.
-            var cookieList = document.cookie.split(';');
-            for (var cInd = 0; cInd < cookieList.length; cInd++) {
+            var cookieList = document.cookie.split(";");
+            for (var cInd = 0; cInd < cookieList.length; cInd ++) {
```
Comment (bytorbrynden) [inline src/generalPurpose.js:15]: ...why the space in between `cInd` and the increment-operator?

Comment (MrMagma) [inline src/generalPurpose.js:15]: It's just a bad habit of mine. I can fix it easily if you like.

**Comment by bytorbrynden** [comment]
Thanks for the pull-request @JavascriptFTW!

**Pull request merged**